File: 02_EstructurasControl/Repetitivas/While/Python/Minijuego_anemia.py
puntos = 0
opcion = ""

while opcion != "3":
    print("\n=== MENU DEL JUEGO ===")
    print("1. Preguntas")
    print("2. Completar palabras")
    print("3. Resultados y salir")
    opcion = input("Elige una opción: ")

    if opcion == "1":
        print("\n--- PREGUNTAS ---")
        print("1) ¿Qué nutriente falta comúnmente en la anemia?")
        print("1. Calcio")
        print("2. Hierro")
        print("3. Sodio")
        respuesta = input("Elige la opción correcta: ")

        if respuesta == "2":
            print("¡Correcto!")
            puntos += 1
        else:
            print("Incorrecto")

    elif opcion == "2":
        print("\n--- COMPLETAR PALABRAS ---")
        print("A_EMI_")
        print("Elige la opción correcta:")
        print("1. A-O")
        print("2. N-A")
        print("3. D-M")
        respuesta = input("Tu respuesta: ")

        if respuesta == "2":
            print("¡Correcto!")
            puntos += 1
        else:
            print("Incorrecto")

    elif opcion == "3":
        print("\n--- RESULTADOS ---")
        print(f"Tu puntaje total es: {puntos}")
        print("¡Gracias por jugar!")

    else:
        print("Opción no válida. Intenta nuevamente.")

File: 02_EstructurasControl/Repetitivas/While/CSharp/Minijuego_anemia.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Net.NetworkInformation;
using System.Text;
using System.Threading.Tasks;

namespace ConsoleApp2
{
    internal class Program
    {
        static void Main(string[] args)
        {
            string opcion = "";
            int puntos = 0;

            while (opcion != "3")
            {
                Console.WriteLine("\n=== MENU DEL JUEGO ===");
                Console.WriteLine("1. Preguntas");
                Console.WriteLine("2. Completar palabras");
                Console.WriteLine("3. Resultados y salir");
                Console.Write("Elige una opción: ");
                opcion = Console.ReadLine();

                if (opcion == "1")
                {
                    Console.WriteLine("\n--- PREGUNTAS ---");
                    Console.WriteLine("1) ¿Qué nutriente falta comúnmente en la anemia?");
                    Console.WriteLine("1. Calcio");
                    Console.WriteLine("2. Hierro");
                    Console.WriteLine("3. Sodio");
                    Console.Write("Elige la opción correcta: ");
                    string respuesta = Console.ReadLine();

                    if (respuesta == "2")
                    {
                        Console.WriteLine("¡Correcto!");
                        puntos++; 
                    }
                    else
                    {
                        Console.WriteLine("Incorrecto");
                    }
                }
                else if (opcion == "2")
                {
                    Console.WriteLine("\n--- COMPLETAR PALABRAS ---");
                    Console.WriteLine("A_EMI_");
                    Console.WriteLine("Elige la opción correcta:");
                    Console.WriteLine("1. A-O");
                    Console.WriteLine("2. N-A");
                    Console.WriteLine("3. D-M");
                    Console.Write("Tu respuesta: ");
                    string respuesta = Console.ReadLine();

                    if (respuesta == "2")
                    {
                        Console.WriteLine("¡Correcto!");
                        puntos++;
                    }
                    else
                    {
                        Console.WriteLine("Incorrecto");
                    }
                }
                else if (opcion == "3")
                {
                    Console.WriteLine("\n--- RESULTADOS ---");
                    Console.WriteLine($"Tu puntaje total es: {puntos}");
                    Console.WriteLine("¡Gracias por jugar!");
                }
                else
                {
                    Console.WriteLine("Opción no válida. Intenta nuevamente.");
                }
            }
        }
    }   
}
